Dockerfile

```diff
@@ -127,6 +131,11 @@ RUN /opt/ort/bin/import_proxy_certs.sh && \
     mkdir -p $GOPATH/bin && \
     curl -ksS https://raw.githubusercontent.com/golang/dep/v$GO_DEP_VERSION/install.sh | sh && \
     curl -ksS https://raw.githubusercontent.com/commercialhaskell/stack/v$HASKELL_STACK_VERSION/etc/scripts/get-stack.sh | sh && \
+    # Install SDKs required for analysis.
+    curl -Os https://dl.google.com/android/repository/commandlinetools-linux-${ANDROID_SDK_VERSION}_latest.zip && \
+    unzip -q commandlinetools-linux-${ANDROID_SDK_VERSION}_latest.zip -d $ANDROID_HOME && \
+    rm commandlinetools-linux-${ANDROID_SDK_VERSION}_latest.zip && \
+    yes | $ANDROID_HOME/tools/bin/sdkmanager --sdk_root=$ANDROID_HOME "platform-tools" && \
```

Comment:
Maybe make it verbose. Maybe add some more packages, like:

`yes | $ANDROID_HOME/tools/bin/sdkmanager --sdk_root=$ANDROID_HOME --verbose "platform-tools" "extras;android;m2repository" "extras;google;m2repository" "ndk-bundle"`

Reply:
I've basically copied the Android SDK installation as-is from `.travis.yml`, so I'd prefer to keep this aligned, and eventually change both at a later time.

Reply:
It's fully up to you. I remembered myself spending some time on figuring this out, so I just wanted to share it, so that you can remember it in case you have an issue.